_data/simple-icons.json

```diff
@@ -10114,7 +10120,7 @@
         {
             "title": "Wipro",
             "hex": "341C53",
-            "source": "https://www.wipro.com/content/dam/nexus/en/service-lines/applications/latest-thinking/state-of-cybersecurity-report-2019.pdf"
+            "source": "https://www.wipro.com/"
```

Comment:
Not sure for this one since there are no vector versions available on their homepage

Reply:
A PNG is not as representative of the icon added since it uses a vector version available in the PDF file as source.

Reply:
should I use the PNG as source and recreate the icon, or should I try and find an unofficial SVG?

Reply:
The best would be to find an official source using the SVG or linking to a PDF that contains one